src/features/dashboard/Chart/Chart.tsx

```diff
@@ -171,10 +171,10 @@ export default () => {
                 },
 
                 panZoom: {
-                    onLiveChanged: live => dispatch(setLive(live)),
+                    onLiveChanged: (live: boolean) => dispatch(setLive(live)),
                     onRangeChanged: (
-                        relativeRange,
-                        referenceNrfmlTimestamp
+                        relativeRange: { min: 0; max: 0 },
```

Comment:
Why `0` and not the more general `number`?

```suggestion
                        relativeRange: { min: number; max: number },
```